### PR TITLE
docs(configuration.md): clarify the explanations about some temp labe…

### PR DESCRIPTION
**Pull request by machine424**
…ls and format for better visibility.

fixes https://github.com/prometheus/prometheus/issues/13857 (or clarifies)

### CODE REVIEW

**Review by SuperQ — DISMISSED** [review]

File: docs/configuration/configuration.md [hunk 1]
```diff
@@ -3264,13 +3264,17 @@ in the configuration file.
 Initially, aside from the configured per-target labels, a target's `job`
 label is set to the `job_name` value of the respective scrape configuration.
 The `__address__` label is set to the `<host>:<port>` address of the target.
-After relabeling, the `instance` label is set to the value of `__address__` by default if
-it was not set during relabeling. The `__scheme__` and `__metrics_path__` labels
-are set to the scheme and metrics path of the target respectively. The `__param_<name>`
-label is set to the value of the first passed URL parameter called `<name>`.
+After relabeling, the `instance` label is set to the value of `__address__` by default if
```
Comment (bboreham) [inline docs/configuration/configuration.md:3267]: stray space at end of line?

**Review by bboreham — APPROVED** [review]
Thanks!

**Comment by bboreham** [comment]
Note DCO check did not pass.

**Comment by machine424** [comment]
I'll go take a walk.

**Pull request merged**